### DISCUSSION TITLE
Update fails via Web

### DISCUSSION THREAD

**Discussion by root9191** [JulianPrieber/littlelink-custom#340] [Issues]
Moin,
so the Update via Web fails everytime. I think since Version 3.x this is happening:
![image](https://user-images.githubusercontent.com/47309435/222219948-486dee5c-ca03-467e-871b-5a111e15f1c8.jpeg)

I updated manually but can you fix this?

**Comment by JulianPrieber** [comment]
Moin!

There are only two instances where this error can appear. What you can try is enable skip update backup in your config and see if that fixes it.

The updater checks before running the update if the backup worked, if it failed it displays an error.

The other option is if this `$updater->source()->isNewVersionAvailable()` fails. If this is the case, then I probably won't be able to help.

**Comment by root9191** [comment]
Moin Moin :)
When i start the Backup manually it works without error. So i think this must be 2nd case.
